ImportExtension/AddImportOperation.swift

```diff
@@ -12,9 +12,9 @@ import AppKit
 fileprivate struct AddImportOperationConstants {
     
     /// Import matchers
-    static let objcImport = ".*#.*(import|include).*[\",<].*[\",>]"
-    static let objcModuleImport = ".*@.*(import).*.;"
-    static let swiftModuleImport = ".*(import) +.*."
+    static let objcImport = "#(import|include).*[\",<].*[\",>]"
```

Comment:
Hey @ClementPadovani, I think this should be `#.*(import|include).*[\",<].*[\",>]`, because one can write
`#     import "Import.h"`.

Reply:
@markohlebar I updated the regex to be `#\s*(import|include).*[\",<].*[\",>]` (see https://github.com/markohlebar/Import/pull/28/commits/2a8e2c38dfab2c45ed0a5e5b9c55d44e59e9570c).